src/Microsoft.DotNet.Wpf/redist/PresentationNative/PresentationNative.vcxproj

```diff
@@ -42,41 +48,28 @@
     <Keyword>Win32Proj</Keyword>
   </PropertyGroup>
   <Import Project="$(VCTargetsPath)\Microsoft.Cpp.targets" />
-  <Target Name="DiscoverPresentationNativeSourcePath"
-          DependsOnTargets="CppHelper"
-          BeforeTargets="CopyRedistFile">
+  <Target Name="DiscoverPresentationNativeSourcePath" DependsOnTargets="CppHelper" BeforeTargets="CopyRedistFile">
     <!-- @(NuGetReferencesForCpp) will have the full paths for PresentationNative.dll and PresentationNative.pdb -->
-    
     <ItemGroup>
       <_PresentationNative Include="$(TargetName)" />
     </ItemGroup>
-    <FilterItem1ByItem2 Item1="@(NuGetReferencesForCpp->'%(FileName)')"
-                        Item2="@(_PresentationNative)"
-                        Metadata2="Identity"
-                        PreserveItem1Metadata="true">
+    <FilterItem1ByItem2 Item1="@(NuGetReferencesForCpp->'%(FileName)')" Item2="@(_PresentationNative)" Metadata2="Identity" PreserveItem1Metadata="true">
       <Output ItemName="_PresentationNativeFiles" TaskParameter="Result" />
     </FilterItem1ByItem2>
-
     <ItemGroup>
       <_PresentationNativeFilePaths Include="@(_PresentationNativeFiles->'%(FilePath)')" />
-      
-      <_PresentationNativeDll Include="@(_PresentationNativeFilePaths)"
-                              Condition="'%(_PresentationNativeFilePaths.Extension)'=='.dll'"/>
-      
-      <_PresentationNativePdb Include="@(_PresentationNativeFilePaths)"
-                              Condition="'%(_PresentationNativeFilePaths.Extension)'=='.pdb'"/>
+      <_PresentationNativeDll Include="@(_PresentationNativeFilePaths)" Condition="'%(_PresentationNativeFilePaths.Extension)'=='.dll'" />
```

Comment:
nit: Can you revert the changes in the later half of the file? They were a bit more readable.